packages/graphql-hooks/package.json

```diff
@@ -35,6 +35,7 @@
     "react": "^17.0.0"
   },
   "dependencies": {
+    "@aws-crypto/sha256-browser": "^2.0.1",
```

Comment:
does this work on the server? this package should also work in SSR scenarios

Reply:
Using `@aws-crypto/sha256-universal` would be a better choice as @radomird pointed out previously

Reply:
Agreed - `@aws-crypto/sha256-universal` seems to be the appropriate choice here.